src/wasm.h

```diff
@@ -1427,15 +1427,15 @@ class Function {
   }
 };
 
+enum class Kind {
```

Comment:
`Kind` is pretty general and not particularly clear... This is just used for imports and exports, right? Could this be called something like `ExternKind`?